internal/osinfo/osinfo_darwin.go

```diff
@@ -3,22 +3,22 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016 Datadog, Inc.
 
-package tracer
+package osinfo
 
 import (
 	"os/exec"
 	"runtime"
 	"strings"
 )
 
-func osName() string {
+func OSName() string {
 	return runtime.GOOS
 }
 
-func osVersion() string {
+func OSVersion() string {
```

Comment:
These functions need to be documented if we are going to export them from the package, even if they *are* internal.

Reply:
> These functions need to be documented if we are going to export them from the package, even if they are internal.

Where does this requirement come from?

If it's just for making the lint CI check happy, we could also look into making that less strict for internal packages.

Personally I think it's a bit odd that moving some code from one internal location to another internal location changing the requirements for documentation 🙈

Reply:
It's not about moving them. They've been renamed starting with an uppercase letter so that they are now exported from the package they are in. 

In fact, I *don't* want to have different requirements for documentation based on any location. We've always tried to make sure that all exported symbols from any package have documentation. I *thought* it was listed in our style guide, but it looks like it's not and is just enforced by the linter. I can add it to the guide.

It doesn't need to be paragraphs, but adding a small note about what it is and why shouldn't be controversial I think.

Reply:
🙇  Thanks, Nick.